ui/shared/LogoMenu.jsx

```diff
@@ -1,39 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {
+  ButtonDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from 'reactstrap';
 
 const choices = [
   { url: '/', text: 'Treeherder' },
   { url: '/perf.html', text: 'Perfherder' },
-  { url: '/intermittent-failures.html', text: 'Intermittent Failures' },
+  { url: '/intermittent-failures.html', text: 'Intermittent Failures View' },
 ];
 
-export default function LogoMenu(props) {
-  const { menuText, menuImage } = props;
-
-  const menuChoices = choices.filter(choice => choice.text !== menuText);
-
-  return (
-    <span className="dropdown">
-      <button
-        id="th-logo"
-        title="Treeherder services"
-        data-toggle="dropdown"
-        className="btn btn-view-nav dropdown-toggle"
-        type="button"
-      >
-        <img src={menuImage} alt={menuText} />
-      </button>
-      <ul className="dropdown-menu" role="menu" aria-labelledby="th-logo">
-        {menuChoices.map(choice => (
-          <li key={choice.text}>
-            <a href={choice.url} className="dropdown-item">
-              {choice.text}
-            </a>
-          </li>
-        ))}
-      </ul>
-    </span>
-  );
+export default class LogoMenu extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      dropdownOpen: false,
+    };
+  }
+
+  toggle() {
+    this.setState({
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
+
+  render() {
+    const { menuText, menuImage } = this.props;
+
+    const menuChoices = choices.filter(choice => choice.text !== menuText);
+    return (
+      <ButtonDropdown isOpen={this.state.dropdownOpen} toggle={this.toggle}>
```

Comment:
Should this be a dropdown instead of a button dropdown?

Also, if this were made an `UncontrolledDropdown` then this component could stay as a functional one without state (reducing the size of the changes required in this PR):
https://reactstrap.github.io/components/dropdowns/#Uncontrolled-Dropdown